docs/topics/con_spring-web-reactive.adoc

```diff
@@ -0,0 +1,19 @@
+[id='spring-web-reactive_{context}']
+= Spring Web
+
+The `spring-web` module provides the foundational elements of the reactive capabilities of  Spring WebFlux, including:
```

Comment:
extra space between of and Spring WebFlux.

Maybe: The `spring-web` module provides the reactive capabilities of  Spring WebFlux, including: